Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -508,6 +508,17 @@ open class BasicFormat: SyntaxRewriter {
       return token
     }
 
+    if token.text == "\n" && token.leadingTrivia.isEmpty && token.trailingTrivia.isEmpty {
```

Comment:
I'm wondering whether it would be more beneficial to use [`isNewline`](https://developer.apple.com/documentation/swift/character/isnewline) instead of `"\n"`. What do you think about this?

This can be achieved in a variety of ways. For example, we could use `text.allSatisfy(\.isNewline)`. However, if we are prioritizing performance, we could aim for O(1) complexity with an approach like the following:
```swift
extension String {
  func isNewline() -> Bool {
    (first?.isNewline ?? false) && dropFirst().isEmpty
  }
}
```

Reply:
I just wanted to say thanks for your review, @Matejkob. Those are valuable comments, they only got overridden by the fact that I found bigger design-ish improvements in the PR.